Applications/Games/SMITE/application.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name"        : "SMITE",
+    "id"          : "smite",
+    "description" : "Join over 25 million SMITE players! Wield Thor’s hammer, transform your foes to stone as Medusa, or choose from the other 90+ mythological icons. You’ll always have more to discover with free updates every two weeks. Enter the Battleground of the Gods!"
+}
```

Comment:
new line at EOF

Reply:
Fixed in https://github.com/PhoenicisOrg/scripts/pull/814/commits/df4b213e88ae0b0a7350698900ddfb42fff74e6c